src/keystats/util.ts

```diff
@@ -1,22 +1,37 @@
 import { KeyStats } from './types'
 
-const numberFormat = (num: number) =>
+export const numberFormat = (num: number) => (
```

Comment:
I forgot to check if `num` is undefined/null when I was writing the function (happens for some companies), which will cause the application to crash if it is. Can you add a check here to see if `num` is null/undefined and return something like `No Data` if it is (with a simple test case)? Or add a todo here for me to do later.